Client/Frontend/Browser/ImageCache/WebImageCache.swift

```diff
@@ -64,12 +64,6 @@ final public class WebImageCache: ImageCacheProtocol {
     let imageOperation = webImageManager.loadImage(with: url, options: SDWebImageOptions(webImageOptions), context: webImageContext, progress: progressBlock) { image, data, error, webImageCacheType, _, imageURL in
       let key = self.webImageManager.cacheKey(for: url)
       if let image = image {
-        let maxSize = CGFloat(FaviconMO.maxSize)
```

Comment:
Are we still using the `load` method here ? Maybe we can remove it all together. As far as I see we were only using this for fav_icons before.

Reply:
It is used in some `MetadataParser` file.